Update font-size-and-colors.md (#732)
Orange-OpenSource/a11y-guidelines: src/en/articles/font-size-and-colors.md

- What is the minimum font size to respect?
- What color can I use for the text? 

## What do the guidelines say?

The [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) do not impose a minimum size or colors for texts. However, two criteria must be taken into consideration:
- <a href="https://www.w3.org/TR/WCAG22/#resize-text"> 1.4.4 Resize text </a>
- <a href="https://www.w3.org/TR/WCAG22/#contrast-minimum"> 1.4.3 Contrast (Minimum) </a> 

### Text size enlargement

If the guidelines do not specify a minimum size for characters, [criterion 1.4.4](https://www.w3.org/TR/WCAG22/#resize-text) indicates that users must be able to increase text size up to 200% without loss of content or functionality. To comply with this criterion, it is essential to test. The procedure for increasing text size is available on the following page: [Text Size Enlargement](https://a11y-guidelines.orange.com/en/web/toolbox/methods-and-test-tools/text-zoom/).

> Sometimes the sizing of certain blocks of text, in particular using sizes in pixels, can lead to loss of information (truncated or overlaped texts), it is therefore advisable to use relative units (%, em, rem...). 

Reviewer: ```suggestion
Sometimes, sizing certain blocks of text, especially using pixel sizes, can lead to loss of information (truncated or overlapped texts), it is therefore advisable to use relative units such as %, em, rem, etc. 
```

> The criterion mentions a few exceptions subject to no constraint, in particular the texts of inactive elements (a button appearing grayed out for example), the texts forming part of a logo or a brand name. 

Reviewer: ```suggestion
The criterion mentions a few exceptions subject to no constraint, in particular the texts of inactive elements (e.g., a grayed-out button), the texts forming part of a logo or a brand name. 
```

## What about users?

The guidelines do not impose a minimum size because it assumes that it is possible to enlarge the text if necessary. In fact, users do not always adjust the size of the text to their needs, due to a lack of habit or ignorance of the possibilities offered to them. This is why the default size must be sufficient to ensure reading comfort.

> The size and the color are not the only characteristics that come into play on the readability of a text, indeed, the typeface or the use of text in italics can have important consequences. You will find some additional recommendations on this subject in the [editorial content section ](https://a11y-guidelines.orange.com/en/content-and-communication/).

Reviewer: ```suggestion
The size and the color are not the only characteristics that come into play on the readability of a text, indeed, the choice of typeface or the use of italics can significantly affect readability. Additional recommendations can be found in the [editorial content section ](https://a11y-guidelines.orange.com/en/content-and-communication/).
```
